R/S3_print.R

```diff
@@ -132,6 +133,11 @@ print.summary.lh_robust <- function(x,...){
   print(summary(x$lm_robust))
   cat("\n\n$linearHypothesis \n \n")
   x <- x[[2]]
-  print(x$linearHypothesis)
+
+  print(attr(x, "linearHypothesis"))
 }
+#' @export
+print.summary.lh <- function(x, ...){
```

Comment:
If you are just calling the default method, you don't even need this specific generic.